### ISSUE TITLE
Unable to run the app in every way

### ISSUE THREAD

**Issue by nicolasalberti00** [chapar-rest/chapar#50]
**Describe the bug**
The app refuses to open, I tried downloading the release on my Mac and it doesn't open, I tried building it from source, following the guide on your website and installing the dependencies, but I get a panic error: "panic: runtime error: slice bounds out of range [:52] with capacity 0".

**To Reproduce**
1. Download the app from the releases and run it
2. Clone the repo and follow the instructions to build it

**Expected behavior**
I expect that the app opens

**Desktop (please complete the following information):**
 - OS: MacOS
 - Version: 14.5
 - Go version: 1.22.4
 - GioUI version: latest available at the moment of writing

**Additional context**
The stack trace of the error shows that the last line of your code called is this one: chapar/ui/app/header.go:155; then the trace goes all the way up to this method from _go-text/typesetting: chapar/vendor/github.com/go-text/typesetting/opentype/tables/glyphs_glyf_src.go:50_ where the function called is: _github.com/go-text/typesetting/opentype/tables.ParseGlyf_, that returns the panic error.

I tried the app also on a Windows PC downloading the release and I had no problem whatsoever, perhaps I was thinking I was missing some fonts and I installed the ones you put inside fonts folder you provide on my Mac.

I would really like to test the app and, if possible, improve it, since I would like to change from Postman's slowness.

Feel free to ask any additional info or troubleshooting steps, I will try them as soon as possible.

Keep up with the good job!

**Comment by mirzakhany** [comment]
Hi @nicolasalberti00 Thanks for reporting this. Im trying to reproduce it but so far no chance. Im running it on a MacOS M2 with no issues. both release and building from source works fine. can you try another Mac or different arch? 

I would be happy to help and understand what's the issue. 🙏

**Comment by nicolasalberti00** [comment]
Hi @mirzakhany , thank you for the fast reply! 
I am sorry I didn't mention I am using an M1 Pro Mac and I did try the Arm64 architecture.
I will try tomorrow with another Mac to see if it runs, I will let you know what I find out

**Comment by nicolasalberti00** [comment]
Hi, I am back with some good news: I tried on another Mac and worked without any problem, I tried running the self-built executable with sudo privileges and it worked, so I suggest there is a permission problem on my Mac, maybe because it's not mine but it's from the company I work in. 
I was not able to run the app from release, perhaps also trying to give the app all the permissions did not work.

I will try it as is for now, hope this experience will be helpful in future!

**Comment by mirzakhany** [comment]
Thanks for letting me know. Im guessing it something with the signature of the app. in Makefile and after building the app Im trying to sign it. you can do the same by running `make build_macos`. 

Hope this solves the issue. but please let me know if you found anything else 🙏

**Comment by nicolasalberti00** [comment]
Hi! Sorry for the late reply, I retried with the method you suggested, so I built it by using the `go build` command described in the documentation, then I ran `make build_macos` and tried to open the generated app, but this did not work, resulting in the same error. 
Anyway, when I have some spare time I'll try to dig in more into this problem, I'll keep you updated, thanks again for the kind reply!

P.S: I have also some suggestion for features, I would like to try and give you a hand to implement those or maybe solving some issues, let me know if you are interested!
